docs/pre_build_command_example.md

```diff
@@ -1,4 +1,4 @@
-Example `build.sh` that uses a secondary `Dockerfile.build` to generate artifacts the main `Dockerfile`
+Example `build_binary.sh` that uses a secondary `Dockerfile.build` to generate artifacts the main `Dockerfile`
```

Comment:
I guess this is a doc for doing things the new way? Though it's not clear about why you would name it `build_binary.sh`, and where you would call it from. We can probably just delete this doc now that we can build scala/java objects on the Samson host itself.